src/controllers/confirmation.controller.ts

```diff
@@ -15,7 +15,7 @@ const createMissingError = (item: string): Error => {
 };
 
 const route = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
-  const companyNum: string = await sessionService.getCompanyInContext(req.chSession);
+  const companyNum: string = sessionService.getCompanyInContext(req.chSession);
```

Comment:
have you run the unit tests? this causes a failure for me

Reply:
Seen this now and fixed it.